README.md

```diff
@@ -17,8 +17,75 @@
+In the case where there is both evidence for a split-read alignment and inter-read jump, the split-read alignment evidence is favored since it gives a precise breakpoint.
+At locations where these jumps occur, breakpoints are marked and the breakpoints are given a unique ID based on the positions of the breakends and the directionality of the left and right strands leading into each breakend.
+
+This process creates a collection of candidate breakpoint locations.
+The output of this tool is a metrics file tabulating the breakpoints and a BAM file with each breakpoint-supporting alignment having custom tags that indicate which breakpoint the alignment supports.
+
+### `fgsv AggregateSvPileup`
+
+Merges nearby pileups of reads supporting putative breakpoints.
+
+```console
+fgsv AggregateSvPileup \
+    --bam sample.bam \
+    --input sample.svpileup.txt \
+    --output sample.svpileup.aggregate.txt
+```
+
+Because of variability in typical short-read alignments, evidence for a single breakpoint may span a few loci near the true breakend loci. For example, if the breakpoint only has intra-read evidence, then the breakpoint could coincidentally occur within the unobserved bases between read 1 and read 2 in a pair. In other cases and due to sequence similarity or homology between each breakend locus, it is not always possible to locate the exact nucleotide point where the breakends occur, and instead a plausible region may exist that supports either breakend loci.
+
+The tool [`fgsv AggregateSvPileup`](https://github.com/fulcrumgenomics/fgsv/blob/main/docs/tools/AggregateSvPileup.md) is used to coalesce nearby breakpoints into one event if they appear to belong to one true breakpoint.
+This polishing step preserves true positive breakpoint events and intends to reduce the number of false positive breakpoint events.
+
+Adjacent breakpoints are only merged if their left breakends map to the same reference sequence, their right breakends map to the same reference sequence, the strandedness of the left and right aligned sub-segments is the same, and their left and right genomic breakend positions are both within a given length threshold.
+
+One shortcoming of the existing behavior, which should be corrected at some point, is that intra-read breakpoint evidence is considered similarly to inter-pair breakpoint evidence even though intra-read breakpoint evidence often has nucleotide-level alignment resolution and inter-pair breakpoint evidence does not.
+
+The output of this tool is a metrics file tabulating the coalesced breakpoints with all previous breakpoint IDs listed for the new breakpoint event and an estimation of the allele frequency of the event based on the alignments that support the breakpoint.
```

Comment:
```suggestion
The output of this tool is a table of coalesced breakpoints. Each aggregate breakpoint is annotated with the constituent pileup IDs and an estimation of the allele frequency of the breakpoint based on the alignments that support the breakpoint.
```

How is allele frequency calculated? Especially without the original bam?

Reply:
More detailed docs exist at the tool-level including a definition for allele frequency:

https://github.com/fulcrumgenomics/fgsv/blob/fbd4861da131e78aec3d64f96a92f34b7144720d/src/main/scala/com/fulcrumgenomics/sv/tools/AggregateSvPileup.scala#L37-L39